rfcs/text/0038/event.yml

```diff
@@ -0,0 +1,48 @@
+# Licensed to Elasticsearch B.V. under one or more contributor
+# license agreements. See the NOTICE file distributed with
+# this work for additional information regarding copyright
+# ownership. Elasticsearch B.V. licenses this file to you under
+# the Apache License, Version 2.0 (the "License"); you may
+# not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# 	http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+---
+- name: event
+  title: Event
+  group: 2
+  short: Fields breaking down the event details.
+  description: >
+    The event fields are ... # Omitted for brevity
+  type: group
+
+  fields:
+    - name: kind
+      level: core
+      type: keyword
+      short: The kind of the event. The highest categorization field in the hierarchy.
+      description: >
+        This is one of four ECS ... # Omitted for brevity
+      example: alert
+      allowed_values:
+        - name: asset
+          description: >
+            This value indicates events whose primary purpose is to store an inventory of
+            assets/entities and their attributes. Assets/entities are objects that are
```

Comment:
I think it should be clearer here about what `objects` fall under our definition of `asset`. Users and devices are implied from the examples later, but I think it's best our definition be explicit.

We can take it even further and clarify this value is reserved and to be used by features of the Elastic stack, like [`signal`](https://www.elastic.co/guide/en/ecs/current/ecs-allowed-values-event-kind.html#ecs-event-kind-signal).